bin/maf/scimaf_dir

```diff
@@ -21,21 +21,34 @@ if __name__ == "__main__":
 
     if args.db is None:
         if os.path.isfile('trackingDb_sqlite.db'):
-            os.remove('trackingDb_sqlite.db')
+            shutil.move('trackingDb_sqlite.db', 'trackingDb_sqlite_o.db')
         db_files = glob.glob('*.db')
     else:
         db_files = [args.db]
     run_names = [os.path.basename(name).replace('.db', '') for name in db_files]
 
     for filename, name in zip(db_files, run_names):
+        outDir = name + '_sci'
+        # Clobber output directory if it exists
+        if os.path.isdir(outDir):
+            shutil.rmtree(outDir)
         opsdb = db.OpsimDatabase(filename)
         colmap = batches.ColMapDict()
-        if os.path.isdir(name + '_sci'):
-            shutil.rmtree(name + '_sci')
+        resultsDb = db.ResultsDb(outDir=outDir)
+        # Set up the metricBundles
         bdict = batches.scienceRadarBatch(runName=name)
-        resultsDb = db.ResultsDb(outDir=name + '_sci')
-        group = mb.MetricBundleGroup(bdict, opsdb, outDir=name + '_sci', resultsDb=resultsDb, saveEarly=False)
+        # Run them, including generating plots
+        group = mb.MetricBundleGroup(bdict, opsdb, outDir=outDir, resultsDb=resultsDb, saveEarly=False)
         group.runAll(clearMemory=True, plotNow=True)
         resultsDb.close()
+        try:
```

Comment:
I don't think this try block is ever going to execute since we don't have config tables anymore.

Reply:
We don't have the 'info' table? Where is any of the information about the version of the inputs that produced the simulation stored?

Reply:
The mafUtils.writeConfigs is a utility which calls the fetchConfig method on the OpsimDatabase and then writes the version information into the output MAF directory along with some additional information on when MAF was run and what date it was run on -- creating the Config files that go into those maf output directories.
Maybe copying the info table isn't needed -- it was more useful with opsim3 where you could extract a subset of the 'most useful' information from the Configuration table in the database, but now there really isn't a short set of 'most useful' information lines. If that version information isn't copied, it does imply a greater need to keep the actual survey.database file (which we do anyway), but in theory, the information in the Config files is intended to 
(A) let you identify the precise simulation the MAF outputs were run on, including the version of opsim/input files
(B) track which version of MAF was run to create the output metric data.
The second one of those isn't tracked anywhere else, so is worth preserving.Suppose we could start writing MAF version and date into each of the metric output files, but we're not even keeping those files much of the time, except for the summary statistic information. So - I guess the alternative would be to write the MAF version and date into the MAF metric data files and also into the resultsDb (and if neither of those are created, then the data is considered truly ephemeral and not necessarily repeatable).

Reply:
Anyway -- it seems like maybe deciding on a better way to keep this information would be useful, but that this is fine for now.

Reply:
Ah, the docstring on fetchConfig is outdated. Something changed in the migration that broke fetchConfig and it doesn't have a unit test.